pkgs/development/mobile/gomobile/default.nix

```
@@ -43,19 +42,25 @@ buildGoModule {
   postInstall = ''
     mkdir -p $out/src/golang.org/x
     ln -s $src $out/src/golang.org/x/mobile
-    wrapProgram $out/bin/gomobile \
+  '';
+
+  fixupPhase = ''
```

We shouldn't overwrite the entire phase.

```suggestion
  postFixup = ''
```

---

Good point.